tests/integration/test_debug.py

```diff
@@ -268,7 +272,29 @@ def test_agentless(monkeypatch):
     tracer = ddtrace.Tracer()
     info = debug.collect(tracer)
 
-    assert info.get("agent_url", "AGENTLESS")
```

Comment:
... so this assert never did anything before?

Reply:
yup! 🤦